_docs/platforms/docker/install.md

```diff
@@ -22,4 +61,3 @@ nano hosts
 ansible-playbook -i hosts site.yaml
 ```
 
-{% include envvars.md %}
```

Comment:
My preference is that wherever we're showing commands that use envvars, we provide an explanation of what they are.  The include seemed like a good way of doing this - not sure the motivation behind removing it.

Reply:
The PR moves the environment variable stuff to the reference section as a first class citizen. It feels neater that way and (IMHO) more intuitive. I can add a note to the various install sections to explain it but it feels superfluous...